codalab/rest/bundles.py

```diff
@@ -467,12 +468,12 @@ def _add_bundle_location(bundle_uuid: str):
     """
     check_bundles_have_all_permission(local.model, request.user, [bundle_uuid])
     need_bypass = query_get_bool('need_bypass', default=False)
-    is_dir = query_get_bool('is_dir', default=False)
+    is_dir = query_get_bool('is_dir', default=None)
```

Comment:
Why this change?

Reply:
Originally we only use this API for `cl upload`, where by default the `is_dir` is set according to the actual file/ folder the user uploaded.
Now we reuse the API for `cl make [bundle_id]  --store`, the user might make multiple bundles into 1 bundle, or make a bundle contains a folder. We can not know `is_dir` in advance, so by default we set it to `None`.

Reply:
But how do we validate `is_dir` then, particularly if the user is using `cl upload`? Shouldn't it be ultimately set to True or False once we create the new bundle?

Reply:
I see!